### PR TITLE
Final Project

### PR DESCRIPTION
**Pull request by hussam-i-am**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,10 +4,145 @@
     <meta name='viewport' content='width=device-width, initial-scale=1.0;'>
     <title >Mercury</title>
     <link rel='stylesheet' href='styles/reset.css'>
+    <link rel="stylesheet" href="https://storage.googleapis.com/code.getmdl.io/1.0.4/material.indigo-pink.min.css">
+	<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```
Comment (hghazzi) [inline index.html]: Fix indentation.

File: index.html [hunk 2]
```diff
@@ -4,10 +4,145 @@
...
+				<button class = "btn-primary form__signin__button" > <a class="form--white" href="#user">Sign In<a></button>
+			</div>
+	 </div> <!--End of form -->
+
+	</script>
+
+	<script id="signup" type="underscore/template">
+		<!-- This is the start of the sign up form -->
+
+		<div class ='form'>
```
Comment (hussam-i-am) [inline index.html:27]: change to `class='form'`

File: javascript/views/signup.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+	function _onLogIn( error, authData ) {
+
+		if ( error ) {
+			// do something in the DOM
+
+			return;
+		}
+console.log( authData )
```
Comment (hussam-i-am) [inline javascript/views/signup.js]: remove console.log

**Comment by hussam-i-am** [comment]
You should fix the indentation of all your files. If you're using tabs, replace them all with 4 spaces and make the files consistent.

**Pull request closed without merging**